mlbench_core/models/pytorch/gnmt/attention.py

```diff
@@ -0,0 +1,176 @@
+import math
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from torch.nn.parameter import Parameter
+
+
+class BahdanauAttention(nn.Module):
+    """
+    Bahdanau Attention (https://arxiv.org/abs/1409.0473)
+    Implementation is very similar to tf.contrib.seq2seq.BahdanauAttention
+
+    Args:
+        query_size: feature dimension for query
```

Comment:
`Args:` in docstrings should specify their type in brackets like `count (int): The count for something, Default: '0'`, this is missing in this file